device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientConfig.java

```diff
@@ -25,8 +27,9 @@
 @Slf4j
 public final class DeviceClientConfig
 {
-    /** The default value for readTimeoutMillis. */
-    private static final int DEFAULT_READ_TIMEOUT_MILLIS = 240000;
+    private static final int DEFAULT_HTTPS_READ_TIMEOUT_MILLIS = 240000;
```

Comment:
4 minute? would you mind adding a comment to that effect

Reply:
sure thing